core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -567,7 +566,10 @@ def validate(self):
                 return
 
             # parse the start-of-file comment lines
-            self.processTopLines(top_comments)
+            if not self.processTopLines(top_comments):
+                self.logger.info(
+                    'Invalid header comments, skipped data in file')
```

Comment:
And top-of-file comments with certain metadata do occur in MAF files (and in other filetypes that may become supported in the future), but as far as I know cBioPortal currently ignores them, so they do not need to be validated. I still believe having a hook to do so could be useful though, so that Validator.validate() never needs to be overridden.